ind_rnn_cell.py

```diff
@@ -7,7 +7,7 @@
 from tensorflow.python.layers import base as base_layer
 
 
-class IndRNNCell(rnn_cell_impl._LayerRNNCell):
+class IndRNNCell(rnn_cell_impl.RNNCell):
```

Comment:
I am new to custom RNN development. Can you explain, why you changed it to `RNNCell`?

Reply:
Sorry, I did a quick search and found that _LayerRNNCell is the way to go, I changed it to make it work with my slightly older install of TF. Will revert.